packages/typegpu/src/core/vertexLayout/vertexAttribute.ts

```diff
@@ -1,4 +1,4 @@
-import type { LooseArray, LooseStruct } from '../../data/dataTypes';
+import type { TgpuDisarray, TgpuUnstruct } from '../../data/dataTypes';
```

Comment:
went with `TgpuDisarray`, but I could be convinced for `Disarray` and `WgslDisarray`

Reply:
I say we go with `Disarray` and `Unstruct` typenames 🧘‍♂️

Reply:
dan